airbyte-integrations/connectors/source-zendesk-chat/integration_tests/integration_test.py

```diff
@@ -0,0 +1,61 @@
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+
+import json
+from pathlib import Path
+
+import pytest
+from airbyte_protocol import ConfiguredAirbyteCatalog, Type
+from base_python import AirbyteLogger
+from source_zendesk_chat.source import SourceZendeskChat
+
+BASE_DIRECTORY = Path(__file__).resolve().parent.parent
+
+
+@pytest.fixture(name="config_credentials")
+def config_credentials_fixture():
+    with open(str(BASE_DIRECTORY / "secrets/config.json"), "r") as f:
+        return json.load(f)
+
+
+@pytest.fixture(name="configured_catalog")
+def configured_catalog_fixture():
+    return ConfiguredAirbyteCatalog.parse_file(BASE_DIRECTORY / "sample_files/agent_timeline_catalog.json")
+
+
+class TestZendeskChatSource:
```

Comment:
we don't need this kind of tests if we use acceptance tests, please check HubSpot repo for example